jax.lax.nextafter is inconsistent with np.nextafter for float64

PR #2393 changes jax.test_util.tolerance to return 0 if given an explicit 0 argument (instead of using a default tolerance). It appears that [a test for `jax.lax.nextafter`](https://github.com/google/jax/blob/ffa03403c9c6d9f071125cdc793c9b88e421cf13/tests/lax_test.py#L86) might have been passing accidentally by using a default tolerance when it was intended to use a tolerance of exactly 0; after the changes in that PR, there are some inconsistencies between the `lax` and `numpy` versions:

```
__________ LaxTest.testOpAgainstNumpyNextafter_float64[3]_float64[3] ___________
[gw0] linux -- Python 3.6.10 /home/travis/miniconda/bin/python
self = <lax_test.LaxTest testMethod=testOpAgainstNumpyNextafter_float64[3]_float64[3]>
op_name = 'nextafter', rng_factory = <function rand_default at 0x7f3fa5c38400>
shapes = ((3,), (3,)), dtype = <class 'numpy.float64'>, tol = 0

...

_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
a = array([5.2921576, 1.2004718, 2.9362137], dtype=float32)
b = array([5.29215704, 1.20047163, 2.93621395]), atol = 0, rtol = 0
    def _assert_numpy_allclose(a, b, atol=None, rtol=None):
      a = a.astype(onp.float32) if a.dtype == dtypes.bfloat16 else a
      b = b.astype(onp.float32) if b.dtype == dtypes.bfloat16 else b
      kw = {}
      if atol: kw["atol"] = atol
      if rtol: kw["rtol"] = rtol
>     onp.testing.assert_allclose(a, b, **kw)
E     AssertionError: 
E     Not equal to tolerance rtol=1e-07, atol=0
E     
E     Mismatched elements: 2 / 3 (66.7%)
E     Max absolute difference: 6.12090903e-07
E     Max relative difference: 1.15660004e-07
E      x: array([5.292158, 1.200472, 2.936214], dtype=float32)
E      y: array([5.292157, 1.200472, 2.936214])
```
PR #2393 disables those tests for now.

--- a/tests/lax_test.py
+++ b/tests/lax_test.py
@@ -199,9 +199,9 @@ class LaxTest(jtu.JaxTestCase):
         for dtype in rec.dtypes)
       for rec in LAX_OPS))
   def testOpAgainstNumpy(self, op_name, rng_factory, shapes, dtype, tol):
-    if op_name == "nextafter" and dtype == onp.float64:
-      raise SkipTest("nextafter inconsistent for float64: "
-                     "https://github.com/google/jax/issues/2403")
+    if (not FLAGS.jax_enable_x64 and op_name == "nextafter"
+        and dtype == onp.float64):
+      raise SkipTest("64-bit mode disabled")
     rng = rng_factory()
     args_maker = lambda: [rng(shape, dtype) for shape in shapes]
     op = getattr(lax, op_name)

